peepingtom/core/datablocks/__init__.py

```diff
@@ -1,7 +1,8 @@
-from .images import ImageBlock
-from .lines import LineBlock
-from .orientations import OrientationBlock
-from .particles import ParticleBlock
-from .points import PointBlock
-from .properties import PropertyBlock
-from .spheres import SphereBlock
+from .imageblock import ImageBlock
+from .lineblock import LineBlock
+from .orientationblock import OrientationBlock
+from .particleblock import ParticleBlock
+from .pointblock import PointBlock
+from .propertyblock import PropertyBlock
+from .sphereblock import SphereBlock
+from .meshblock import MeshBlock
```

Comment:
These should also be changed everywhere else (depictors, peeper, tests...)

Reply:
everywhere else uses imports which derive from this as far as I can see! 
i.e.
`import peepingtom.core.datablocks.ImageBlock`
so this doesn't cause any problems